closure/goog/net/jsloader.js

```diff
@@ -103,7 +113,7 @@ goog.net.jsloader.loadMany = function(uris, opt_options) {
   //
   // To work around this issue, all module loads share a queue.
   if (!uris.length) {
-    return;
+    return goog.async.Deferred.succeed(null);
```

Comment:
Actually, I'm guessing you're trying to emulate what `load` does? If so, that's fine.